### PR TITLE
Show LoginPopup in comment textarea via focus/change events

### LINKED ISSUE #1182: Login/Signup prompt does not appear if you tab into a comment form
If you tab into the comment form, this popup doesn't show up but typing in the field is still disabled so it just looks like the site is broken.

### LINKED ISSUE #1228: Login button on LoginPopup has red text on red background
screen shot 2018-09-20 at 8 56 06 am

### PR DESCRIPTION
**Pull request by mbertrand**
#### Pre-Flight checklist

- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Fixes #1182
Fixes #1228

#### What's this PR do?
- Shows the LoginPopup in the comment textarea via onChange and onFocus events instead of onClick.
- Fixes CSS for the LoginPopup login button

#### How should this be manually tested?
- Go to a public post detail page as an anonymous user
- Use the tab key to navigate to the comment form.  The LoginPopup should appear you tab into it.
- Close the popup, then click inside the textarea.  The popup should appear again.

### CODE REVIEW

**Comment by pdpinch** [comment]
Do we expect these events will work for mobile? 

I guess onFocus() should fire if you’re clicking, tabbing or tapping.

**Comment by mbertrand** [comment]
Tested with iPhone X simulator & it works:

screen shot 2018-09-20 at 8 53 22 am

**Comment by aliceriot** [comment]
works great! reading code now

File: static/js/components/CommentForms.js [hunk 1]
```diff
@@ -72,6 +72,14 @@ const getPostReplyInitialValue = (parent: Post) => ({
   text:    ""
 })
 
+const userConditionalFunction = (
```
Comment (aliceriot) [inline static/js/components/CommentForms.js]: can we do a more description name for this? just something that captures the internal logic a little better

**Pull request merged**